torch/autograd/profiler.py

```diff
@@ -1437,6 +1479,18 @@ def add_column(padding, text_dir='>'):
         headers.append('Source Location')
         add_column(src_column_width, text_dir='<')
 
+    if with_flops:
+        # Auto-scaling of flops header
+        US_IN_SECOND = 1000.0 * 1000.0  # self_cpu_time_total is in us
+        raw_flops = []
+        for evt in events:
+            if evt.flops != 0:
+                evt.flops = float(evt.flops) / evt.self_cpu_time_total * US_IN_SECOND
+                raw_flops.append(evt.flops)
+        (flops_scale, flops_header) = auto_scale_flops(min(raw_flops))
```

Comment:
we should prob. also check just in case that raw_flops is not empty

Reply:
if it's empty - e.g. we couldn't find any valid flops, then just we assume that with_flops is false